src/models/hooks/activityReport.js

```diff
@@ -155,7 +155,7 @@ const propogateSubmissionStatus = async (sequelize, instance, options) => {
   }
 };
 
-const determineObjectiveStatus = async (activityReportId, sequelize) => {
+const determineObjectiveStatus = async (activityReportId, sequelize, isResetToDraft) => {
```

Comment:
Small nitpick: should this actually be called "isUnlocked"? Reset to draft is the button you press when a report is submitted and then un-submitted by the user,  unlock is the button you click when you have an approved report and are setting it to draft, and the if statement is checking for the latter case.

Reply:
👍